### PR TITLE
[mediaplayer] Update for iOS 10.3 beta 1

### PR DESCRIPTION
**Pull request by VincentDondain**

### CODE REVIEW

**Comment by VincentDondain** [comment]
For:

``` C#
[Export ("insertQueueDescriptor:afterItem:")]
void InsertAfter (MPMusicPlayerQueueDescriptor queueDescriptor, [NullAllowed] MPMediaItem item);
```

Guess we could also go `Insert` but I find that it's not explicit enough.
Also we might eventually get a `insertQueueDescriptor:beforeItem:`

File: src/MediaPlayer/MediaPlayer.cs [hunk 1]
```diff
@@ -303,6 +303,7 @@ public enum MPErrorCode : nint {
 		NotSupported,
 		[iOS (10,1)]
 		Cancelled,
+		RequestTimedOut,
```
Comment (dalexsoto) [inline src/MediaPlayer/MediaPlayer.cs]: Missing `[iOS (10,3)]`?

Comment (spouliot) [inline src/MediaPlayer/MediaPlayer.cs]: @dalexsoto no, it's an error (output) enum so it's not needed. The one on `Cancelled` should be removed (not critical).

Comment (VincentDondain) [inline src/MediaPlayer/MediaPlayer.cs]: Yes that's why I didn't add it.
Didn't want to remove the other one to not add noise.

**Comment by VincentDondain** [comment]
Unrelated test failure:
- https://bugzilla.xamarin.com/show_bug.cgi?id=52162 - monotouch-test - Unified 32bits

**Pull request merged**